### ISSUE TITLE
Only export openapi.json, not openapi.yaml

### ISSUE THREAD

**Issue by blairj09** [rstudio/plumber#615]
Currently, an API built against #598 published to RStudio Connect results in the following:
```
2020/07/21 20:31:22.911759238 Using Packrat dir /opt/rstudio-connect/mnt/app/packrat/lib/x86_64-pc-linux-gnu/3.6.0
2020/07/21 20:31:23.477196448 `$run(swagger)` has been deprecated in v1.0.0 and will be removed in a coming release. Please use `$setUI(ui)`
2020/07/21 20:31:23.477397750 Running plumber API at http://127.0.0.1:38837
2020/07/21 20:31:23.484268239 Error in value[[3L]](cond) : 
2020/07/21 20:31:23.484278045   yaml must be installed for the yaml serializer to work
2020/07/21 20:31:23.484308581 Calls: local ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
2020/07/21 20:31:23.484309648 Execution halted
```

This is because we serialize both `openapi.json` and `openapi.yaml` - see: https://github.com/rstudio/plumber/pull/598/files#diff-4f632064d1ef846baad1bfd3b7fc8dffR111

I think we only need to serialize `openapi.json` and remove the potential dependency on `yaml`

**Comment by meztez** [comment]
Yes this is the reason, I agree with @blairj09.

**Comment by meztez** [comment]
Fixed/Merged in #598
